geonode/myjson.py

```diff
@@ -0,0 +1,30 @@
+import json
+from django.contrib.gis.geos import GEOSGeometry
+from django.contrib.gis.geos import Polygon
+
+
+class MyEncoder(json.JSONEncoder):
+
+    def default(self, obj):
+        if isinstance(obj, Polygon):
+            return {
+                '__type__':'__polygon__',
+                'ewkt': obj.ewkt
+            }
+        else:
+            return json.JSONEncoder.default(self, obj)
```

Comment:
@GeoTob I will have a look of this on Wed, following is the error targetting this line.
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dingdingfan/geonode_p3_manual_testing/venv/lib/python3.6/site-packages/kombu/serialization.py", line 50, in _reraise_errors
    yield
  File "/home/dingdingfan/geonode_p3_manual_testing/venv/lib/python3.6/site-packages/kombu/serialization.py", line 221, in dumps
    payload = encoder(data)
  File "/home/dingdingfan/geonode_p3_manual_testing/geonode/geonode/myjson.py", line 26, in my_dumps
    return json.dumps(obj, cls=MyEncoder)
  File "/usr/lib/python3.6/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/usr/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/dingdingfan/geonode_p3_manual_testing/geonode/geonode/myjson.py", line 15, in default
    return json.JSONEncoder.default(self, obj)
  File "/usr/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'Decimal' is not JSON serializable
```